Tests/CoreTests/StringExtensionsTests.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  StringExtensionsTests.swift
+//  plank
+//
+//  Created by Michael Schneider on 6/16/17.
+//
+//
+
+import XCTest
+
+@testable import Core
+
+class StringExtensionsTests: XCTestCase {
+
+    func testUppercaseFirstCharacter() {
+        XCTAssert("plank".uppercaseFirst == "Plank")
+    }
+
+    func testSuffixSubstring() {
+        XCTAssert("plank".suffixSubstring(2) == "nk")
+    }
+
+    func testSnakeCaseToCamelCase() {
+        XCTAssert("created_at".snakeCaseToCamelCase() == "CreatedAt")
+        XCTAssert("CreatedAt".snakeCaseToCamelCase() == "CreatedAt")
+    }
+
+    func testSnakeCaseToPropertyName() {
+        XCTAssert("created_at".snakeCaseToPropertyName() == "createdAt")
+
+        // (@maicki): This test would currently fail as the result is "CreatedAt". 
+        // XCTAssert("CreatedAt".snakeCaseToPropertyName() == "createdAt")
```

Comment:
I think what you have here is the desired output. I'm assuming we generate `createdat` right now?